tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "outDir": "lib",
     "rootDir": "src",
     "resolveJsonModule": true,
+    "skipLibCheck": true,
```

Comment:
What does this fix for you?

Reply:
It fixes an issue with TS attempting to compile using types within node_modules, which is not what we want. Specifically there's an issue with eslint's types.